code/modules/clothing/head/jobs.dm

```diff
@@ -81,36 +80,47 @@
 
 /obj/item/clothing/head/beret/sec/navy/hos
 	name = "Commander beret"
-	desc = "Grey beret with a Commander's rank emblem. For commanders that are more inclined towards style than safety."
+	desc = "Grey beret with a Lieutenant's rank emblem. For officers that are more inclined towards style than safety."
 	icon_state = "beret_navy_hos"
 	spawn_blacklisted = TRUE
 
 /obj/item/clothing/head/beret/sec/navy/warden
 	name = "Sergeant beret"
-	desc = "Red beret with a Sergeant's rank emblem. For sergeants that are more inclined towards style than safety."
+	desc = "Red beret with a Sergeant's rank emblem. For Sergeants that are more inclined towards style than safety."
 	icon_state = "beret_navy_warden"
 	spawn_blacklisted = TRUE
 
+/obj/item/clothing/head/beret/purple
+	name = "Purple beret"
+	desc = "A stylish, if purple, beret."
+	icon_state = "beret_purple"
+
 /obj/item/clothing/head/beret/engineering
-	name = "engineering beret"
-	desc = "A beret with the engineering insignia emblazoned on it. For engineers that are more inclined towards style than safety."
+	name = "Technomancer yellow beret"
+	desc = "A beret with the Technomancer leauge insignia emblazoned on it. For engineers that are more inclined towards style than safety."
```

Comment:
```suggestion
	desc = "A beret with the Technomancer League insignia emblazoned on it. For engineers that are more inclined towards style than safety."
```

Reply:
oh god it wont let me commit suggestion and the checks are failing